fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1233,7 +1234,9 @@ private ExtractedParameterValue processPrimitiveValue(FHIRPathSystemValue system
             parameter = p;
         } else if (systemValue.isTemporalValue()) {
             DateParmVal p = new DateParmVal();
-            p.setValueDate(Timestamp.from(QueryBuilderUtil.getInstantFromPartial(systemValue.asTemporalValue().temporal())));
+            TemporalAccessor v = systemValue.asTemporalValue().temporal();
+            java.time.Instant inst = DateTimeHandler.generateValue(v);
+            p.setValueDate(DateTimeHandler.generateTimestamp(inst));
```

Comment:
is this used anywhere?  do we have a test for it?

Reply:
also, are there any places left that use QueryBuilderUtil's datetime helpers?  if not, make it they can be removed?

Reply:
Yes, I have a full suite of tests for DateTimeHandler. 
I can check.  I wanted to be consistent with the timestamp/instant.

Reply:
I've cleaned this up to remove QueryBuilderUtil's time handler/test so we're consistent